package.json

```diff
@@ -14,15 +14,9 @@
     "generate": "graphql-codegen --config codegen.yml"
   },
   "dependencies": {
-    "@apollo/react-components": "^3",
-    "@apollo/react-hoc": "^3",
-    "@apollo/react-hooks": "^3",
-    "@apollo/react-testing": "^3",
+    "@apollo/client": "^3.3.7",
```

Comment:
Since we are using whole number `^N` versions in the rest of our package.json, can you turn this into version into a whole number?